client/modules/reconportal/src/ReconSidePanel/ReconSidePanel.tsx

```diff
@@ -25,30 +25,12 @@ import {
 import { formatDate } from '@client/workspace';
 import dayjs from 'dayjs';
 import { CloseOutlined } from '@ant-design/icons';
-
-const EVENT_TYPE_COLORS = {
-  earthquake: '#e46e28',
-  flood: '#4285F4',
-  tsunami: '#a765fe',
-  landslide: '#62a241',
-  hurricane: '#d34141',
-  tornado: '#9100ff',
-};
-
-const markerIconUrls = {
-  earthquake:
-    'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-orange.png',
-  flood:
-    'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-blue.png',
-  tsunami:
-    'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-violet.png',
-  landslide:
-    'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-green.png',
-  hurricane:
-    'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-red.png',
-  tornado:
-    'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-violet.png',
-};
+import {
+  EVENT_TYPE_COLORS,
```

Comment:
is possible to drop import of EVENT_TYPE_COLORS. might no longer be needed here as using getREconEVentCOlor